src/helpers.js

```diff
@@ -82,9 +81,8 @@ export async function getDependencies(packageName: string): Promise<Array<Depend
 }
 
 export async function promptUser(packageName: string, dependencies: Array<Dependency>): Promise<'Yes' | 'No' | 'Never'> {
-  const configPath = Path.join(atom.getConfigDirPath(), 'package-deps-state.json')
-  const configFile = await ConfigFile.get(configPath, { ignored: [] }, { createIfNonExistent: true })
-  const ignoredPackages = await configFile.get('ignored')
+  const configPath = atom.config.getUserConfigPath()
+  const ignoredPackages = atom.config.get('package-deps.ignored')
```

Comment:
Can we make with `sb-package-deps.ignored` instead? I don't want it to conflict in case someone actually publishes a `package-deps` named package